### ISSUE TITLE
Dynamic init gmap

### ISSUE THREAD

**Issue by freenetwork** [xkjyeah/vue-google-maps#621]
I want to dynamically transfer the API key from the admin panel, how to do it?

**Comment by tsworman** [comment]
@freenetwork - From the Google Maps Admin panel? Do you just not want to include the API key in your source code? If so, I'd look into using environment variables and loading it at that way at build time.

**Comment by diegoazh** [comment]
Yes, using environment variables you can do it. I think that is the better option.
Read this [vue documentation](https://cli.vuejs.org/guide/mode-and-env.html#environment-variables)

**Comment by notflip** [comment]
Found a way to do it.

Install the plugin without specifying options

```
Vue.use(VueGoogleMaps);
```

In your component
```
beforeMount () {
    VueGoogleMaps.loadGmapApi({
      key: 'your-g-api-key,
      libraries: 'places'
    })
  },
```

**Comment by eliecer2000** [comment]
There is an example in the following link that could help you. [here!](https://github.com/xkjyeah/vue-google-maps/issues/135)
